atom/atom.py

```diff
@@ -194,7 +203,7 @@ def __new__(meta, name, bases, dct):
         post_validates = []         # Post validate methods: _post_validate_*
         seen_sentinels = set()      # The set of seen sentinels
         seen_decorated = set()      # The set of seen @observe decorators
-        for key, value in dct.iteritems():
+        for key, value in dct.items():
```

Comment:
I know you mentioned this in the intro for this PR, but I thought I would ask again: If the original version was using `iteritems`, don't you think it's a good idea to preserve that behavior?

I intend for this comment to cover all instances of `iteritems`->`items` and `itervalues`->`values`, but I'll spare you the repeated callouts.